### ISSUE TITLE
Missing label

### ISSUE THREAD

**Issue by felixzapata** [googlecodelabs/your-first-pwapp#76]
There is no label element in the dialog container for the cities, so the "Lighthouse" tool will throw an error of accessibility.

**Comment by petele** [comment]
Hi!

Thanks for the feedback. I've rewritten the codelab from scratch and believe I have fixed this issue. The code has been updated, and the new [guide](https://g.co/codelabs/pwa) is now live.

If you find any issues with the new codelab, please let us know!

Thanks!
